docs/core/macos-prerequisites.md

```diff
@@ -36,7 +36,7 @@ ln -s /usr/local/opt/openssl/lib/libcrypto.1.0.0.dylib /usr/local/lib/
 ln -s /usr/local/opt/openssl/lib/libssl.1.0.0.dylib /usr/local/lib/
 ```
 
-After installing OpenSSL, obtain the official [.NET Core SDK for Mac installer](https://go.microsoft.com/fwlink/?linkid=843444). .NET Core 1.1.1 is the latest version. For long-term support versions and additional downloads, visit [.NET Downloads: All downloads](https://www.microsoft.com/net/download/core). If you have problems with the installation on macOS, consult [Known issues & workarounds](https://github.com/dotnet/core/blob/master/cli/known-issues.md).
+After installing OpenSSL, obtain the official [.NET Core SDK for Mac installer](https://go.microsoft.com/fwlink/p/?linkid=843444). .NET Core 1.1.1 is the latest version. For long-term support versions and additional downloads, visit [.NET Downloads: All downloads](https://www.microsoft.com/net/download/core). If you have problems with the installation on macOS, consult [Known issues & workarounds](https://github.com/dotnet/core/blob/master/cli/known-issues.md).
```

Comment:
By reading this more carefully, I realize that we should just link to the https://www.microsoft.com/net/download/core page instead of having a direct link to the installer. Otherwise, for every release, we'll have to be in the business of updating this page. We also don't need to state what's the latest version here.